internal/service/alarms/internal/alertmanager/alertmanager.go

```diff
@@ -63,3 +68,101 @@ func Setup(ctx context.Context, cl client.Client) error {
 	slog.Info("Successfully configured alertmanager")
 	return nil
 }
+
+// ConvertAmToAlarmEventRecordModels get alarmEventRecords based on the alertmanager notification and AlarmDefinition
+func ConvertAmToAlarmEventRecordModels(am *api.AlertmanagerNotification, aDefinitionRecords []models.AlarmDefinition, clusterIDToResourceTypeID map[uuid.UUID]uuid.UUID) []models.AlarmEventRecord {
```

Comment:
This goes better in the db/models/converters.go

Reply:
yeah originally thought that but its cyclical import error (would have to move all the Am conversion code there as as well)